spring-cloud-deployer-spi-test/pom.xml

```diff
@@ -59,8 +57,7 @@
 		</dependency>
 		<dependency>
 			<groupId>org.hamcrest</groupId>
-			<artifactId>hamcrest-all</artifactId>
-			<version>${hamcrest.version}</version>
+			<artifactId>hamcrest-library</artifactId>
```

Comment:
hamcrest-all is hamcrest + hamcrest-core + hamcrest-library in unzipped single jar
BUT 
hamcrest-library includes hamcrest + hamcrest-core